cmd/minishift/cmd/start.go

```diff
@@ -238,9 +238,16 @@ func runStart(cmd *cobra.Command, args []string) {
 		proxyConfig.ApplyToEnvironment()
 	}
 
-	// preflight checks (after start)
+	// preflight checks and set static-ip (after start)
 	if viper.GetString(configCmd.VmDriver.Name) != genericDriver {
 		preflightChecksAfterStartingHost(hostVm.Driver)
+		fmt.Print("-- Writing current configuration for static assignment of IP address ... ")
+		_, err := minishiftNetwork.ConfigureStaticAssignment(hostVm.Driver)
+		if err != nil {
+			fmt.Println("FAIL")
```

Comment:
@gbraad @praveenkumar Somehow we should communicate this failure as a warning as it does not effect Minishift to work normally unless IP changes. I am not sure how at this point of time but will suggest something once I review the PR fully.

Reply:
@LalatenduMohanty @gbraad We can just put "WARN" instead of "FAIL" if that what you think would be good, I don't want to have any other phrase than OK, FAIL, SKIP and WARN.

Reply:
WARN seems more appropriate